tests/Unit/PointwiseFunctions/GeneralRelativity/Test_Ricci.cpp

```diff
@@ -7,123 +7,136 @@
 #include "tests/Unit/TestHelpers.hpp"
 
 namespace {
-void test_1d_spatial_ricci(const DataVector& used_for_size) {
+template <typename DataType>
+void test_1d_spatial_ricci(const DataType& used_for_size) {
   constexpr size_t spatial_dim = 1;
   const auto ricci = gr::ricci_tensor(
-      make_spatial_christoffel_second_kind<spatial_dim>(0.),
-      make_deriv_spatial_christoffel_second_kind<spatial_dim>(0.));
-  CHECK(ricci.get(0, 0) == approx(0.));
-
-  check_tensor_doubles_equals_tensor_datavectors(
-      gr::ricci_tensor(
-          make_spatial_christoffel_second_kind<spatial_dim>(used_for_size),
-          make_deriv_spatial_christoffel_second_kind<spatial_dim>(
-              used_for_size)),
-      ricci);
+      make_spatial_christoffel_second_kind<spatial_dim, DataType>(
```

Comment:
remove `, DataType`

Reply:
and repeat for all the places you did that

Reply:
Sure. Please explain why this is necessary?

Reply:
The real question is, is having it there necessary?